addons/mass_mailing/models/mass_mailing.py

```diff
@@ -620,6 +621,15 @@ def onchange_mass_mailing_campaign_id(self, cr, uid, id, mass_mailing_campaign_i
                    'medium_id': mass_mailing_campaign[0].medium_id.id}
             return {'value': dic}
 
+    def create(self, cr, uid, vals, context=None):
+        if not vals['mass_mailing_campaign_id']:
```

Comment:
Is this code necessary ? The ORM will automatically create the campaign (inherits).

And why is Superuser_id the 'user_id' of all campaigns ? Does that mean anything ?